ceph_medic/checks/osds.py

```diff
@@ -0,0 +1,31 @@
+
+
+#
+# Utilities
+#
+
+def get_osd_ceph_fsids(data):
+    fsids = []
+    for file_path in data['paths']['/var/lib/ceph']['files'].keys():
+        if "ceph_fsid" in file_path:
+            fsids.append(data['paths']['/var/lib/ceph']['files'][file_path]['contents'])
+    return set(fsids)
+
+
+def check_osd_ceph_fsid(host, data):
+    code = 'WOSD1'
+    msg = """
+    Multiple ceph_fsid values found in /var/lib/ceph/osd.
```

Comment:
these are usually one-liners, otherwise the terminal output might get too crowded. The full description is fine for the docs